Antonio_Ramos_Documentation

```diff
@@ -494,4 +436,83 @@ I am not installing arcpy because it appears (see line 449) that a previous chan
+
+- trying to understand how counts are created.
+
+9/9/2020
+- I am having trouble replicating the creation of the array which holds these point counts when I try to use simplified data. I believe this may mean there is an issue in the code on lines 130-133 of kfunction_3stage.py. (no issue found in code).
+
+- There may be an issue with the way in which the distance matrix is being calculated. This issue might be on the code side or the data side and is the one I am currently investigating.
+
+- after reviewing the distance matrix through the variable explorer in spyder the data appears to be incorrect and it is largely either 0 or *** and has little to no variation in between
+
+- reviewing how distance matrix is calculated
+
+- no issue with matrix construction fuctinos so the issue must stem from the data being fed into the function
+
+- issue identified with the way in which the XY coordinates are being imported. For some reason, the coordinates imported do not match the coordinates from the data as seen in arcgis.
+
+- exploring the data import to better understand why the coordinates do not match
+
+- on line 40 of kfunction Local_3Stage.py we define a variable,
+  "pFeatures = gpd.read_file(points).filter(['REF_ID', 'SIC', 'geometry'], axis = 1)".
+  
+- Here "geometry" contains the data for the XY coordinates. However, the data it contains when reviewed in Spyder's variable explorer does not match the data from arcgis that is visible in the data attribute table. 
+
+9/10/2020
+
+- "geometry" is an geopandas array created when a we use geopandas to import a shapefile. The array contains the information for the structure of the geometric objects within the shapefile. This is defined by a shape and coordinates of the shapes centroid or the coordinates of the point if the shape is a point.
+
+- our CA_Lab_Data.shp file has centroid locations saved under the XY coordinates, however, geopandas is constructing its own centroid locations for the geometry index
+
+- could resolve issue by either changing the reference variables on lines 39 77 and 78, or I can try to correct the way in which geopandas is creating geometries for the file.
+
+9/11/2020
+
+- corrected all geometries by changing the code in GoodLabs_CRS_Trans.py. I deleted lines 18 - 21 and replaced it with the following code...
+
+  "# Create GeoDataFrame with new CRS
+     geometry = pFeatures.geometry
+     geo_df2 = gpd.GeoDataFrame(data2, geometry = geometry)
+     geo_df2 = geo_df2.to_crs('epsg:5070') 
+     geo_df2 = geo_df2.set_geometry(geometry)"
+
+- now the CRS is set properly so that when geopandas reads in the file during runtime of K-function Local_3Stage.py it will be able to properly construct the coordinate pairs under geomentry.
+
+- when running program no cluster file is exported and point resutls appear to still be the same as before where we are either getting 0.01 or 1 for the cluster significance
+
+9/12/2020
```

Comment:
Is something missing here at the end?

Reply:
Yeah something didn't get copied right this should've included documentation up until Sep 20th. I think I may have accidentally deleted some of the documentation when I put in the new pull request. I will have rewrite it because it isn't on my computer. Nothing too important though, just some troubleshooting ideas and steps I took to resolve the geometry issues but none of them worked.